engine/runner/src/main/scala/org/enso/runner/Main.scala

```diff
@@ -502,13 +502,19 @@ object Main {
       logLevel,
       logMasking,
       enableIrCaches           = true,
+      strictErrors             = true,
       useGlobalIrCacheLocation = shouldUseGlobalCache
     )
     val topScope = context.getTopScope
-    topScope.compile(shouldCompileDependencies)
-
-    context.context.close()
-    exitSuccess()
+    try {
+      topScope.compile(shouldCompileDependencies)
+      exitSuccess()
+    } catch {
+      case _: Throwable =>
+        exitFail()
```

Comment:
I assume that the compiler already reported the errors, and we don't want to log the exception here as well

Reply:
Since this is our internal tool anyway, I'm thinking if it could be better to log the exception.

In case some other exception than compiler error gets to here, it will now be swallowed. Our life debugging such a hypothetical error will be so much easier if it's logged.

Maybe not full stack trace necessarily, but at least a name of the exception?